h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> Also, we have not patched jschannel. As far as I know we have not patched any vendor libs.
> This is not a practice I have ever tolerated.
> [...]
> Ed Summers got a patch submitted and accepted on JSChannel.

IIRC, we started to use the patched version before it was merged upstream. (See here: https://github.com/hypothesis/h/commit/1f458105bee01e2cb5e999708c23bcebb7ec56b2 )

So first we submitted our patch upstream, and then we started to use it immediately, without waiting for upstream to merge. In the meantime, we used a patched version of the vendor lib. (Although, we didn't call the directory `vendor` back then.)

When submitting the current PR, we were exactly in the same situation now: I submitted the upstream PR, and then started to use the patched version immediately.

Later, in https://github.com/hypothesis/h/commit/f0803a102b12ec970156533fb5e5a1eb10812799, I improved the situation by clearly including the URL of our fork in the source, so that nobody will replace it with an upstream version, even in the unlikely case when upstream would awaken from it's 4-year long slumber.

Can we move on now?